huaweicloud/services/drs/resource_huaweicloud_drs_job.go

```diff
@@ -744,6 +780,23 @@ func resourceJobUpdate(ctx context.Context, d *schema.ResourceData, meta interfa
 		}
 	}
 
+	if d.HasChange("auto_renew") {
+		// resource_id is different from job_id
+		detailResp, err := jobs.Get(client, jobs.QueryJobReq{Jobs: []string{d.Id()}})
```

Comment:
how about add a new function in common package to get resource IDs from an order?